ledger/ledger-api-client/src/it/scala/com/digitalasset/ledger/client/CommandClientIT.scala

```diff
@@ -127,7 +127,7 @@ final class CommandClientIT
   private def readExpectedElements[T](
       src: Source[T, NotUsed],
       expected: Set[T],
-      timeLimit: Span = 3.seconds): Future[(Set[T], Set[T])] =
```

Comment:
Why did you remove the default? Seems like a reasonable thing to keep.

Reply:
It is unused. In that it is like a private definition alongside that is never used, hence `-Ywarn-unused` including checks for these sorts of things.